bindings/web5_c/src/crypto/key_managers/poc.rs

```diff
@@ -0,0 +1,38 @@
+use super::{call_get_signer, call_import_private_jwk, CKeyManager};
+use crate::{
+    c::free_bytes,
+    crypto::{
+        dsa::{call_sign, free_csigner},
+        jwk::{free_cjwk, CJwk},
+    },
+};
+use web5::crypto::jwk::Jwk;
+
+#[no_mangle]
+pub extern "C" fn poc_key_manager_from_foreign(manager: *const CKeyManager) {
```

Comment:
this is a test basically?

Reply:
yup! it proves the injection of a foreign implementation of a key manager